
The following have been reloaded with a version change:
  1) cuda/12.6.0 => cuda/12.1.1


CondaError: Run 'conda init' before 'conda activate'

The following values were not passed to `accelerate launch` and had defaults used instead:
	`--num_processes` was set to a value of `1`
	`--num_machines` was set to a value of `1`
	`--mixed_precision` was set to a value of `'no'`
	`--dynamo_backend` was set to a value of `'no'`
To avoid this warning pass in values for each of the problematic parameters or run `accelerate config`.
wandb: Currently logged in as: wang4538 (rajdeeph-purdue-university). Use `wandb login --relogin` to force relogin
wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: Tracking run with wandb version 0.19.4
wandb: Run data is saved locally in /home/wang4538/DGMS-master/scripts/wandb/run-20250311_020010-uidobqml
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run meta-llama/Llama-3.2-1B_K16_KL_SPAS0.75_temp0.001_LR5e-6_PRTEMP0.01_WD5e-7_SIGMA3
wandb: ‚≠êÔ∏è View project at https://wandb.ai/rajdeeph-purdue-university/SQS_GLUE
wandb: üöÄ View run at https://wandb.ai/rajdeeph-purdue-university/SQS_GLUE/runs/uidobqml
Traceback (most recent call last):
  File "/home/wang4538/DGMS-master/scripts/../glue_training.py", line 600, in <module>
    main(args)
  File "/home/wang4538/DGMS-master/scripts/../glue_training.py", line 328, in main
    InitModel(model, args.sigma)
  File "/home/wang4538/DGMS-master/scripts/../glue_training.py", line 148, in InitModel
    m.init_mask_params(sigma)
  File "/home/wang4538/DGMS-master/QuantAttention.py", line 762, in init_mask_params
    self.q_proj.sub_distribution = gmm_approximation(self.k_level, self.q_proj.weight, self.temperature, init_method, sigma)
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wang4538/DGMS-master/modeling/DGMS/GMM.py", line 307, in gmm_approximation
    return GaussianMixtureModel(num_components, init_weights, temperature, init_method, sigma)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wang4538/DGMS-master/modeling/DGMS/GMM.py", line 69, in __init__
    self.params_initialization(init_weights, init_method)
  File "/home/wang4538/DGMS-master/modeling/DGMS/GMM.py", line 134, in params_initialization
    if sigma_init.isnan().any() or sigma_init.eq(0.0).any():
       ^^^^^^^^^^
UnboundLocalError: cannot access local variable 'sigma_init' where it is not associated with a value
Traceback (most recent call last):
  File "/home/wang4538/miniconda3/envs/LLM/bin/accelerate", line 10, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/wang4538/miniconda3/envs/LLM/lib/python3.12/site-packages/accelerate/commands/accelerate_cli.py", line 48, in main
    args.func(args)
  File "/home/wang4538/miniconda3/envs/LLM/lib/python3.12/site-packages/accelerate/commands/launch.py", line 1172, in launch_command
    simple_launcher(args)
  File "/home/wang4538/miniconda3/envs/LLM/lib/python3.12/site-packages/accelerate/commands/launch.py", line 762, in simple_launcher
    raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)
subprocess.CalledProcessError: Command '['/home/wang4538/miniconda3/envs/LLM/bin/python3.12', '../glue_training.py', '--dataset_name', 'ptb_text_only', '--task_name', 'sst2', '--K', '16', '--tau', '0.001', '--weight_decay', '5e-7', '--prune', '--model_name', 'meta-llama/Llama-3.2-1B', '--lr', '5e-6', '--duration', '5', '--alpha_f', '0.01', '--seed', '428', '--init_method', 'k-means', '--run_name', 'meta-llama/Llama-3.2-1B_K16_KL_SPAS0.75_temp0.001_LR5e-6_PRTEMP0.01_WD5e-7_SIGMA3', '--autoresume', '--eval_interval', '1ep', '--prune_scale', '0.01', '--prune_start', '0.05', '--sigma', '3', '--init_sparsity', '0.0', '--final_sparsity', '0.75', '--prune_end', '0.9', '--optimizer', 'sgd', '--batch_size', '16', '--max_seq_length', '384', '--preprocessing_num_workers', '4', '--save_folder', '/scratch/gilbreth/wang4538/DGMS/Run/GLUE_sst2_meta-llama/Llama-3.2-1B/compressed/']' returned non-zero exit status 1.
